packages/fxa-settings/src/components/UnitRow/index.tsx

```diff
@@ -7,16 +7,59 @@ import classNames from 'classnames';
 import { useFocusOnTriggeringElementOnClose } from '../../lib/hooks';
 import { Link, RouteComponentProps, useLocation } from '@reach/router';
 
+type ModalButtonProps = {
+  ctaText: string;
+  classname?: string;
+  revealModal: () => void;
+  modalRevealed?: boolean;
+  alertBarRevealed?: boolean;
+};
+
+export const ModalButton = ({
+  ctaText,
+  classname,
+  revealModal,
+  modalRevealed,
+  alertBarRevealed,
+}: ModalButtonProps) => {
+  const modalTriggerElement = useRef<HTMLButtonElement>(null);
+  // If the UnitRow children contains an AlertBar that is revealed,
+  // don't redirect focus back to the element that opened the modal
+  // because focus will be set in the AlertBar.
+  useFocusOnTriggeringElementOnClose(
+    modalRevealed,
+    modalTriggerElement,
+    alertBarRevealed
+  );
+
+  return (
+    <button
+      className={classNames(
+        'cta-neutral cta-base transition-standard ml-3',
+        classname
+      )}
+      data-testid="unit-row-modal"
+      ref={modalTriggerElement}
+      onClick={revealModal}
+    >
+      {ctaText}
+    </button>
+  );
+};
+
 type UnitRowProps = {
   header: string;
   headerValue: string | null;
   noHeaderValueText?: string;
   ctaText?: string;
+  secondaryCtaText?: string;
+  secondaryCtaRoute?: string;
+  secondaryButtonClassName?: string;
   children?: React.ReactNode;
   headerValueClassName?: string;
   route?: string;
   revealModal?: () => void;
-  modalRevealed?: boolean;
+  revealSecondaryModal?: () => void;
   alertBarRevealed?: boolean;
```

Comment:
Looks like `alertBarRevealed` is no longer used here.

Reply:
It needs to stay, unfortunately since it is passed through to `ModalButton`

Reply:
Maybe it should be, but currently it's not being used 🤔 
![Screen Shot 2020-08-27 at 1 34 01 PM](https://user-images.githubusercontent.com/6392049/91475747-04a01980-e86a-11ea-8f7d-87cf54c7394a.png)